### PR TITLE
OV-283: modify videos table

### PR DESCRIPTION
**Pull request by sergiy4**
This PR modifies the video table.

### CODE REVIEW

File: shared/src/bundles/videos/types/create-video-request-dto.type.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 type CreateVideoRequestDto = {
     userId: string;
     name: string;
-    url: string;
+    url?: string;
+    composition: string;
+    previewUrl: string;
```
Comment (o-nedashkivska) [inline shared/src/bundles/videos/types/create-video-request-dto.type.ts:6]: I am not sure if we need to send `previewUrl` when creating video. I guess we should define this property based on first scene of the composition that frontend sends (could be done in separate ticket).

Comment (o-nedashkivska) [reply shared/src/bundles/videos/types/create-video-request-dto.type.ts:6]: Also url property will be set only on backend after video rendering and I am not sure we need it in CreateVideoRequestDto. Or should we leave it for testing purposes?

Comment (Sanchousina) [reply shared/src/bundles/videos/types/create-video-request-dto.type.ts:6]: Agree, we don't have url yet when creating video

File: backend/src/bundles/avatar-videos/avatar-videos.service.ts [hunk 2]
```diff
@@ -114,6 +114,9 @@ class AvatarVideoService {
             name: getFileName(id),
             url: savedUrl,
             userId,
+            composition: JSON.stringify({}),
+            previewUrl:
+                'https://speech.microsoft.com/assets/avatar/harry/harry-business-thumbnail.png',
```
Comment (Sanchousina) [inline backend/src/bundles/avatar-videos/avatar-videos.service.ts:119]: Is this mocked data for now? Why not take the image already from the composition?

**Review by sofiia-trokhymchuk — CHANGES_REQUESTED** [review]
Resolve all the comments

File: backend/src/bundles/avatar-videos/avatar-videos.service.ts (hunk 2 above)
Comment (sergiy4) [reply backend/src/bundles/avatar-videos/avatar-videos.service.ts:119]: I've already done this in my next PR.  https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/318/files#diff-277a782f500c4ec4c6a53ff830a0acbb45e0d9f852afa74c0b1c2d25f54eccbeR60

File: shared/src/bundles/videos/types/create-video-request-dto.type.ts (hunk 1 above)
Comment (sergiy4) [reply shared/src/bundles/videos/types/create-video-request-dto.type.ts:6]: I've already solved it in my next PR https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/318

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**